### PR TITLE
Viewswap

### PR DESCRIPTION
**Pull request by comm-scripter**
added work safe links

### CODE REVIEW

File: src/app/components/views/weather-view/weather-view.component.html [hunk 1]
```diff
@@ -9,16 +9,57 @@ <h5>Active weather alerts in your area:</h5>
...
-  <div class="card text-dark bg-light mb-3">
-    <div class="card-header">
-      <h3>{{alert?.event}}</h3>
-    </div>
-    <div class="card-body">
-      <p class="card-text">{{alert?.description}}</p>
-    </div>
-    <div class="card-footer">
-      <h6>{{alert?.severity}}</h6>
+  <div class="alert-card">
```
Comment (micah-wehrle) [inline src/app/components/views/weather-view/weather-view.component.html:12]: select this line down to line 44 and hit shift-tab

File: src/app/components/views/weather-view/weather-view.component.ts [hunk 2]
```diff
@@ -12,9 +12,14 @@ import { WeatherService } from 'src/app/services/weather.service';
 export class WeatherViewComponent implements OnInit, OnDestroy {
   public weatherAlertResponse: WeatherAlertResponse;
   private ngUnsubscribe = new Subject<void>();
+  public showMoreButtonText = true;
```
Comment (looksliketyler) [inline src/app/components/views/weather-view/weather-view.component.ts:15]: data type and a comment as to why its being set to true

File: src/app/components/views/weather-view/weather-view.component.ts [hunk 3]
```diff
@@ -12,9 +12,14 @@ import { WeatherService } from 'src/app/services/weather.service';
...
 
   constructor(private weatherService: WeatherService) { }
 
+  onMoreClicked() {
```
Comment (looksliketyler) [inline src/app/components/views/weather-view/weather-view.component.ts:19]: jsdoc, access modifer and return type

File: src/app/components/views/weather-view/weather-view.component.scss [hunk 4]
```diff
@@ -1,3 +1,23 @@
 .card {
   overflow: hidden;
 }
+.alert-card {
+  width: 100%;
+  background-color: white;
+  padding: 10px;
+}
+
+.equipment-icon-box {
```
Comment (micah-wehrle) [inline src/app/components/views/weather-view/weather-view.component.scss:10]: unused classes, same as the stb ones

File: src/app/components/views/weather-view/weather-view.component.ts [hunk 5]
```diff
@@ -13,6 +13,7 @@ export class WeatherViewComponent implements OnInit, OnDestroy {
   public weatherAlertResponse: WeatherAlertResponse;
   private ngUnsubscribe = new Subject<void>();
 
+
```
Comment (looksliketyler) [inline src/app/components/views/weather-view/weather-view.component.ts:16]:
```suggestion
```

**Review by looksliketyler — APPROVED** [review]

**Comment by micah-wehrle** [comment]
merged a while ago, deleting

**Pull request merged**